nucleus/admin/util/src/main/java/com/sun/enterprise/admin/remote/sse/GfSseEventReceiver.java

```diff
@@ -89,7 +89,7 @@ public GfSseInboundEvent readEvent() throws IOException {
                         } else if(data != '\n') {
                             baos.write(data);
                             currentState = State.FIELD_NAME;
-                        } else if(data == '\n') {
```

Comment:
Are we sure that removing of the If condition here is not causing any issue? I think the logic changes when we remove the condition